File: fsharp/raindrops/Raindrops.fs
module Raindrops
open System

let convert (n: int): string =
  let l = [
    (if n % 3 = 0 then "Pling" else "");
    (if n % 5 = 0 then "Plang" else "");
    (if n % 7 = 0 then "Plong" else "")
  ]
  let s = String.Join("", l)

  if s.Length = 0 then n.ToString() else s

File: python/raindrops/raindrops.py
def convert(number):
    pling_plang_plong = "".join(
      [
          "Pling" if number % 3 == 0 else "",
          "Plang" if number % 5 == 0 else "",
          "Plong" if number % 7 == 0 else ""
      ]
    )
    return pling_plang_plong if len(pling_plang_plong) > 0 else str(number)
